Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN apt-get install -y --no-install-recommends \
     gsfonts \
     latexmk \
     rsync \
+    sudo \
```

Comment:
How was it working without this? Wasn't certbot-auto calling sudo internally? Why didn't it need to be explicitly installed before?

Reply:
I tried to explain this in my PR description where I said:

> The addition of `sudo` in the Dockerfile was needed because `certbot-auto` was previously handling calling `sudo` or not based on whether or not we were already root.

To expand on that though, [certbot-auto only sets up sudo/su to be used if it is needed](https://github.com/certbot/certbot/blob/2d62dec7ecd7f62a77768aa951342b5cbfc06abc/certbot-auto#L189-L195), [it's needed in Travis](https://docs.travis-ci.com/user/installing-dependencies/#installing-packages-on-standard-infrastructure), but technically not needed in the Dockerfile because the build is running as root inside the container. Since `certbot-auto` was already running as `root` in the Docker image, it didn't call `su` or `sudo`.

I added the dependency on `sudo` to the Dockerfile so we could use the same command in both cases for simplicity.

Reply:
Ooooh, I see now, I didn't realize it was running in two different environments, that makes sense.